src/runtime/objmodel.cpp

```diff
@@ -2484,7 +2488,7 @@ Box* callFunc(BoxedFunctionBase* func, CallRewriteArgs* rewrite_args, ArgPassSpe
         }
     }
 
-    std::vector<Box*> varargs;
+    llvm::SmallVector<Box*, 8> varargs;
```

Comment:
Hmm I think this might need to use an StlCompatAllocator -- what if given_varargs is not a container and generates objects on the fly?  I think then they could only be referenced by varargs?